### DISCUSSION TITLE
Adjust the SH-RPI output voltage

### DISCUSSION THREAD

**Discussion by manjem** [hatlabs/discussions#115] [General]
Hi,
I’m running the SH-RPI in an enclosed box with a 5v fan (all bought at the same time). The fan makes a lite of noise. Is it possible to either reduce the output voltage to 3.3 or schedule it on an of during different time of the day? (Eg night time)

**Comment by mairas** [comment]
You can safely plug the fan to the 3.3V pins and it'll run fine, and probably almost silently. Of course, the cooling effect will also be reduced, but any airflow makes a big impact.

Timing or controlling is a bit more difficult. You'd need to create a transistor switch that you control with a GPIO pin.

A CM4 baseboard with a 4-pin fan connector makes the job much easier - Linux has kernel support for controlling the fan without any additional software, and you can even do PWM control for smooth RPM adjustment.
